docs/source/enable/traits.rst

```diff
@@ -0,0 +1,45 @@
+Enable Custom Traits
+====================
+Enable defines several `trait types <https://docs.enthought.com/traits>`_ which
```

Comment:
Maybe link to the custom traits section of the traits docs?
https://docs.enthought.com/traits/traits_user_manual/custom.html

I actually am unsure which would make more sense, so feel free to leave if you prefer the home page

Reply:
I'll leave it like this